jumanji/training/configs/env/tetris.yaml

```diff
@@ -2,15 +2,15 @@ name: tetris
 registered_version: Tetris-v0
 
 network:
-    conv_num_channels: 32
-    tetromino_layers: 16
-    final_layer_dims: [128,128]
+    conv_num_channels: 64
+    tetromino_layers: [16, 16]
```

Comment:
what does "tetromino" mean?

Reply:
Please see the Tetris wiki [here](https://tetris.fandom.com/wiki/Tetromino)

Reply:
Well I feel silly...